### PR TITLE
Main page

### LINKED ISSUE #10: Implement create/join party frontend
Implement main Spartify page in React where users can either choose to create or join parties

### LINKED ISSUE #12: Set up React Router
Route to different pages

### PR DESCRIPTION
**Pull request by apzsfo**
This is a PR for the static main page of our application. Currently, I can't query GraphQL, so the placeholder name and password are "Kathy's Party" and "Kathy's Password", respectively. To open a popup, you can click on either "create a party" or "join a party". To exit a popup, just re-click the same button.

### CODE REVIEW

File: web/src/view/nav/route.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
  */
 export enum Route {
   HOME = 'app/index',
+  PARTY = 'app/party',
   LECTURES = 'app/lectures',
   PROJECTS = 'app/projects',
   PLAYGROUND = 'app/playground',
```
Comment (Rohanator9000) [inline web/src/view/nav/route.ts]: Should we delete these unused routes? In general, I don't know if we need to delete code we're not using.

Comment (apzsfo) [reply web/src/view/nav/route.ts]: I think we can delete them. I don't think there should be any issue with that

**Review by rning — CHANGES_REQUESTED** [review]
Is there a reason we aren't using Storybook?

Can you describe how you tested?

Also, can we delete the unneeded class website tsx files?

**Review by danielskathyd — APPROVED** [review]

**Comment by danielskathyd** [comment]
I agree with deleting the old class website files + routes, but looks good otherwise

**Review by Rohanator9000 — APPROVED** [review]

**Review by rning — APPROVED** [review]
I can't run it, but the deletions look good

**Pull request merged**